packages/Algernon.package/ALGBackground.class/instance/initialize.st

```diff
@@ -1,11 +1,9 @@
 initialization
 initialize
-	
 	super initialize.
-	self
-		color: self class backgroundColor;
-		extent: World extent;
-		center: World center;
-		borderWidth: 0.
+	self color: self class backgroundColor;
+		 extent: World extent;
+		 center: World center;
+		 borderWidth: 0.
 	self currentWorld addMorphFront: self.
-	self hide.
\ No newline at end of file
+	self hide
```

Comment:
Shouldn't there be a dot here?

Reply:
Isn't our policy to omit the dot at the last line ?

Reply:
I think only after returns. But I'm not sure. Just do what you think is right.

Reply:
I will keep it this way and add a note to #136 . 🙂